ludwig/datasets/model_configs/adult_census_income_default.yaml

```diff
@@ -0,0 +1,42 @@
+output_features:
+  - name: income
+    type: category
+input_features:
+  - name: age
+    type: numerical
+  - name: workclass
+    type: category
+  - name: fnlwgt
+    type: numerical
+  - name: education
+    type: category
+  - name: education-num
+    type: numerical
+  - name: marital-status
+    type: category
+  - name: occupation
+    type: category
+  - name: relationship
+    type: category
+  - name: race
+    type: category
+  - name: sex
+    type: category
+  - name: capital-gain
+    type: numerical
+  - name: capital-loss
+    type: numerical
+  - name: hours-per-week
+    type: numerical
+  - name: native-country
+    type: category
+combiner:
+  type: concat
+  num_fc_layers: 3
+  fc_size: 128
+  dropout: 0.1
+training:
+  batch_size: 256
+  learning_rate: .001
+  epochs: 1
+  steps_per_checkpoint: 1
```

Comment:
For default configs, I was thinking that they would only contain `input_features` and `output_features`. The rest would be taken from the defaults we define, and if we find better general defaults than the ones we currently have, we can update those. This way the default config could be a blank canvas for experimentation.

Reply:
I think this is a good goal to iterate towards.  As it stands, training would take too long on some datasets and the default encoders for some datatypes will use a lot of memory.